homeassistant/components/sensor/wunderground.py

```diff
@@ -20,13 +20,15 @@
 
 _RESOURCE = 'http://api.wunderground.com/api/{}/conditions/q/'
 _ALERTS = 'http://api.wunderground.com/api/{}/alerts/q/'
+_FORECAST = 'http://api.wunderground.com/api/{}/forecast/q/'
 _LOGGER = logging.getLogger(__name__)
 
 CONF_ATTRIBUTION = "Data provided by the WUnderground weather service"
 CONF_PWS_ID = 'pws_id'
 
 MIN_TIME_BETWEEN_UPDATES_ALERTS = timedelta(minutes=15)
 MIN_TIME_BETWEEN_UPDATES_OBSERVATION = timedelta(minutes=5)
+MIN_TIME_BETWEEN_UPDATES_FORECAST = timedelta(minutes=15)
```

Comment:
by default, WU allows 500 API calls under the free plan. Using the values above it will generate 480 calls a day which can be a little tight considering some extra calls or resets during the an update or HA restart. 

Since the forecast does not change that often how about to set MIN_TIME_BETWEEN_UPDATES_FORECAST 30 minutes?  

```
288 reqs/day for OBSERVATION
96 reqs/day for ALERTS
48 reqs/day for FORECAST
----
432 reqs/day total
```